database/migrations/2021_12_09_171715_create_sport_activities_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateSportActivitiesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sport_activities', function (Blueprint $table) {
+            $table->id();
+            $table->string('name', 20);
+            $table->text('description');
+            $table->dateTime('active_from');
```

Comment:
Initialy you planned to use date. DateTime will complicate your life fir this columns.

Reply:
I get it, fixed